forc-plugins/forc-debug/src/cli/commands.rs

```diff
@@ -257,7 +316,7 @@ async fn cmd_memory(state: &mut State, mut args: Vec<String>) -> Result<()> {
         .unwrap_or(0);
 
     // Ensure no extra arguments
```

Comment:
- // Ensure no extra arguments
+ // Ensure the argument count is at most 2

Reply:
cool, changed here [20ad867](https://github.com/FuelLabs/sway/pull/6830/commits/20ad86701c849c10fce85a7a095d423bd8da055b)